packages/ui/package.json

```diff
@@ -43,6 +43,9 @@
     "test:open": "cypress open --component",
     "prepublishOnly": "pnpm build"
   },
+  "peerDependencies": {
+    "vitest": ">=0.30.1 <1"
+  },
```

Comment:
Note that this makes the feature a breaking change - which is OK, but should be indicated in release notes. Maybe even in the PR description here.